UT-Alignment-12d-1/readme.md

```diff
@@ -1,34 +1,38 @@
-## Intent
+
+### Intent
 
 IFC 4x3 RC2
 This scenario has a simple alignment with left arc in horizontal and no vertical.
 
-That is it has:
- Civil Horizontal Geometry (HG): straight, an arc with ifc positive radius of curvature, and a straight
+That is it has Civil Horizontal Geometry (HG):
+
+- straight
+- an arc with ifc positive radius of curvature
+- a straight
 
 In the ifc file there is only the ifc semantic definition of the alignment and no ifc geometry.
 The Ifc file was generated by 12d Model. 
 
-## Prerequisites
+### Prerequisites
 
-- none
+- ProjectSetup-1
 
-## Content
+### Content
 
 This scenario covers the additional concepts and/or IFC entities:
 
-- IfcAlignment                  - fudged Axis as there is no Ifc geometry
-- IfcAlignmentHorizontal
-- IfcAlignmentHorizontalSegment - PredefinedType LINE
-- IfcAlignmentHorizontalSegment - PredefinedType CIRCULARARC for a left arc
+- 'IfcAlignment'                  - fudged 'Axis' as there is no Ifc geometry
+- 'IfcAlignmentHorizontal'
+- 'IfcAlignmentHorizontalSegment' with 'PredefinedType=LINE'
+- 'IfcAlignmentHorizontalSegment' with 'PredefinedType=CIRCULARARC' for a left arc
```

Comment:
These are not the correct "code snippet" escape characters in markdown syntax. Apparently it's called [Grave accent](https://en.wikipedia.org/wiki/Grave_accent).